Packs/CortexAttackSurfaceManagement/README.md

```diff
@@ -159,13 +160,20 @@ A playbook that given the IP address enriches Rapid7 information relevant to ASM
 
 A playbook that creates an ASM Remediation Confirmation Scan using an existing service ID, if the scan does not already exist;. It then polls for results of a scan.
 
-![Cortex ASM - Remediation Confirmation Scan](https://raw.githubusercontent.com/demisto/content/ec3ba284d4f60e27928a2dc4db19d8758a36fe98/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Remediation_Confirmation_Scan.png)
+![Cortex ASM - Remediation Confirmation Scan](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Remediation_Confirmation_Scan.png)
+
 #### Cortex ASM - Remediation Guidance
 
 A playbook that pulls remediation guidance off of a list based on ASM RuleID to be used in service owner notifications (email or ticketing system).
 
 ![Cortex ASM - Remediation Guidance](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Remediation_Guidance.png)
 
+#### Cortex ASM - Remediation Objectives
+
+A playbook that populates remediation objectives field uses to display the remediation actions to the end user.
+
+![CCortex ASM - Remediation Objectives]https://raw.githubusercontent.com/demisto/content/5f71853b59431ca60b1b783867b89f819accfefd/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Remediation_Objectives.png)
```

Comment:
NIT: Misspelt Cortex

Reply:
fixed in [pack README fix](https://github.com/demisto/content/pull/27795/commits/ada9998fed431f1f0f03288052f0d69246dca467)